pinodo/coffee-mathcer
# Add eslint v8

### Summary
Add eslint ^8.24.0 to devDependencies
References #21 

### Steps to test
- Remove node_modules folder
- Reinstall all packages by running `npm i` 
- Run `npm run lint` and ensure no errors
